### PR TITLE
Fabo/fix polkadot account creation

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
This is local account creaton + app account creation

File: app/src/components/network/NetworkList.vue [hunk 1]
```diff
@@ -53,10 +53,8 @@ export default {
   },
   methods: {
     async selectNetworkHandler(network) {
-      if (this.isExtension) {
-        if (this.networkId !== network.id) {
-          await this.$store.dispatch(`setNetwork`, network)
-        }
+      if (this.networkId !== network.id) {
+        await this.$store.dispatch(`setNetwork`, network)
```
Comment (Bitcoinera) [inline app/src/components/network/NetworkList.vue:57]: so this wasn't ever needed right? This check I mean

Comment (Bitcoinera) [reply app/src/components/network/NetworkList.vue:57]: For me it works fine both ways :thinking: ?

Comment (faboweb) [reply app/src/components/network/NetworkList.vue:57]: can you rephrase?

Comment (faboweb) [reply app/src/components/network/NetworkList.vue:57]: if you are in cosmos-hub-mainnet and create an account for polkadot, the created account is for cosmos

Comment (Bitcoinera) [reply app/src/components/network/NetworkList.vue:57]: but how can you create an account for Polkadot in cosmos-hub-mainnet? You always select the network first, before creating any account? :question:

Comment (faboweb) [reply app/src/components/network/NetworkList.vue:57]: `setNetwork` was not called. But farther down, `currentNetwork` was used for some properties. This variable still had the old network.
